.ci_support/linux_64_.yaml

```
@@ -1,8 +1,10 @@
 cdt_name:
-- cos6
+- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

This python_min argument should be `3.10` following the pysal guidelines. @jGaboardi 
```suggestion
- '3.10'
```

---

But that is not set here. These files are automatic.

---

I think this can stay as `3.9` here in `.ci_support/linux_64_.yaml`. As long as `3.10` is declared in `recipe/meta.yaml`.

See an example over in the `esda-feedstock`:
* [`.ci_support/linux_64_.yaml#L10`](https://github.com/conda-forge/esda-feedstock/blob/2d58d307a88a7b34dcce08da3d0238d5dcb19db0/.ci_support/linux_64_.yaml#L10)
* [`recipe/meta.yaml#L1`](https://github.com/conda-forge/esda-feedstock/blob/2d58d307a88a7b34dcce08da3d0238d5dcb19db0/recipe/meta.yaml#L1)

---

Oh, I did not see the `recipe/meta.yaml`. My bad!